### PR TITLE
Feat/tier cards

### PR DESCRIPTION
**Pull request by plondon**
## Description
Tiers pages in settings/profile and exchange/profile

## Change Type
Please enter one or more of the following: 
- Feature

## Code Checklist
- [ ] Code compiles successfully (verified via `yarn start`)
- [ ] No lint issues exist (verified via `yarn lint`)
- [ ] New and existing unit tests pass (verified via `yarn test`)
- [ ] `README.md` and other documentation is updated as needed

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+          values={{ time: path([tier, 'time'], TIERS) }}
+        />
+      )
+  }
+}
+
+export const TIERS = {
+  1: {
+    time: '3',
+    isActive: propEq('state', 'ACTIVE'),
```
Comment (tony-blockchain) [inline packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js]: shouldn't this point to current user tier being more than tierIndex?

File: packages/blockchain-wallet-v4-frontend/src/data/modules/profile/model.js [hunk 2]
```diff
@@ -22,6 +22,22 @@ export const TIERS = {
   2: 2
 }
 
+export const INITIAL_TIERS = [
```
Comment (tony-blockchain) [inline packages/blockchain-wallet-v4-frontend/src/data/modules/profile/model.js]: this one's a stub right?

File: packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js [hunk 3]
```diff
@@ -0,0 +1,173 @@
+import React from 'react'
+import { path, prop, propEq } from 'ramda'
+import { FormattedMessage } from 'react-intl'
+import { Text } from 'blockchain-info-components'
+
+export const getTiming = (state, tier) => {
```
Comment (tony-blockchain) [inline packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js]: does not belong to model but to some component messaging service

File: packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+          <FormattedMessage
+            id='components.identityverification.tiercard.annuallimit'
+            defaultMessage='Annual Swap Limit'
+          />
+        </Text>
+      )
+    },
+    requirements: [
+      {
+        message: (
```
Comment (tony-blockchain) [inline packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js]: I'd rather have some constant name in the model and map it to message in component service( constant namings can be reused in different screens with another messages)

File: packages/blockchain-wallet-v4-frontend/src/data/modules/profile/model.js (hunk 2 above)
Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/data/modules/profile/model.js]: Yeah

File: packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js (hunk 3 above)
Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js]: I will move to services.js file

File: packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js (hunk 4 above)
Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js]: Will move to services.js

**Comment by plondon** [comment]
ExchangeProfile and Settings/Profile are the exact same so I will refactor that

File: packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js (hunk 1 above)
Comment (plondon) [reply packages/blockchain-wallet-v4-frontend/src/components/IdentityVerification/TierCard/model.js]: Yes this was a mock I forgot to update

**Comment by tony-blockchain** [comment]
Please add a back button to the /exchange/profile

**Comment by tony-blockchain** [comment]
also I think that Tier cards CTA should be based on /kyc/tiers endpoint: VERIFIED -> Swap, NONE -> Get Started

**Pull request merged**